plugins/overlays/download.js

```diff
@@ -35,10 +34,23 @@ class DownloadOverlay extends React.Component {
       );
   }
 
+  componentizeUserMessage(message){
```

Comment:
doesn't use `this` so it shouldn't be attached to the class, just a normal function